### PR TITLE
Function docs

### PR DESCRIPTION
**Pull request by rich-iannone**
This PR updates function documentation for all of the 'rule' functions found in `rules.R`.

### CODE REVIEW

File: R/rules.R [hunk 1]
```diff
@@ -525,18 +764,43 @@ sv_between <- function(left,
...
+#' }
+#' 
+#' shinyApp(ui, server)
+#' 
+#' }
+#' 
+#' @family rule functions
+#' 
+#' @seealso The [sv_between()] function, which tests whether a field values
+#'   between two boundary values.
```
Comment (jcheng5) [inline R/rules.R:803]: Can we try testing with passing a factor to `sv_in_set` and then using a string value?

**Review by jcheng5 — APPROVED** [review]

**Review by jcheng5 — APPROVED** [review]
